src/test/java/qna/domain/AnswerTest.java

```diff
@@ -1,6 +1,66 @@
 package qna.domain;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
 public class AnswerTest {
     public static final Answer A1 = new Answer(UserTest.JAVAJIGI, QuestionTest.Q1, "Answers Contents1");
     public static final Answer A2 = new Answer(UserTest.SANJIGI, QuestionTest.Q1, "Answers Contents2");
+    @Autowired
+    private AnswerRepository answers;
+
+    @Test
+    void save() {
+        // when
+        final Answer actual = answers.save(A1);
+
+        //then
+        assertThat(answers.findById(actual.getId()).get()).isEqualTo(actual);
+    }
+
+    @Test
+    void findById() {
+        // given
+        final long expected1 = answers.save(A1).getId();
+        final long expected2 = answers.save(A2).getId();
+
+        // when
+        final Answer actual1 = answers.findById(expected1).get();
+        final Answer actual2 = answers.findById(expected2).get();
+
+        // then
+        assertAll(
+                () -> assertThat(actual1.getId()).isEqualTo(expected1),
+                () -> assertThat(actual2.getId()).isEqualTo(expected2)
+        );
```

Comment:
아래의 테스트 코드로 충분하지 않을까요?

```suggestion
        // when
        final Optional<Answer> actual1 = answers.findById(expected1);
        final Optional<Answer> actual2 = answers.findById(expected2);

        // then
        assertAll(
                () -> assertThat(actual1).isPresent(),
                () -> assertThat(actual2).isPresent()
        );
```

Reply:
조회의 테스트인데 제가 찾아올려고 하는 값이 정확히 그 값인지 확인해야 하는 것 아닐까요?